### ISSUE TITLE
ListView Support

### ISSUE THREAD

**Issue by chrisjenx** [chrisjenx/ParallaxScrollView#3]
Add ListView support so the background can scroll in relation to a ListView

**Comment by nadavfima** [comment]
I'd love to see this implemented!

**Comment by chrisjenx** [comment]
Yeah its in the pipeline, just need some time to actually do it :)

**Comment by nadavfima** [comment]
Sick! can't wait!  

## 

Nadav Fima
Sent with Sparrow (http://www.sparrowmailapp.com/?sig)

**Comment by ber4444** [comment]
Any news about this? Maybe an alpha version? You can also send me in a private message as I urgently want to work on it for my current project.

**Comment by chrisjenx** [comment]
I'm going to re write the lib to make it abstract-able so this would be
easier in the future.

But I'm really tied up at the moment. Sorry.

**Comment by moeedayazbutt** [comment]
+1 for implementing ListView.
